src/CMakeLists.txt

```diff
@@ -290,6 +290,13 @@ endif()
 # Set install RPATH
 #
 
+if(SKBUILD)
```

Comment:
this is not actual anymore (see https://github.com/openvinotoolkit/openvino_tokenizers/pull/166)

Reply:
it's valid for release (current target branch), while for master the condition should be different

Reply:
Does that fix something in the release branch? If not then I would like to exclude this fix

Reply:
Removed